src/resource-manager.c

```diff
@@ -1106,8 +1106,13 @@ command_special_processing (ResourceManager *resmgr,
                             Tpm2Command     *command)
 {
     Tpm2Response *response   = NULL;
+    TPM2_CC cc = tpm2_command_get_code (command);
 
-    switch (tpm2_command_get_code (command)) {
+    if (tpm2_command_has_auths(command) && cc == TPM2_CC_GetCapability) {
+      return NULL;
+    }
+
+    switch (cc) {
```

Comment:
I would prefer we keep the handling of commands within the respective switch statement for the command code, and thus would prefer something like:
```diff
diff --git a/src/resource-manager.c b/src/resource-manager.c
index 6459b1f409e6..45fb36bb99a8 100644
--- a/src/resource-manager.c
+++ b/src/resource-manager.c
@@ -1121,8 +1121,10 @@ command_special_processing (ResourceManager *resmgr,
         response = resource_manager_load_context (resmgr, command);
         break;
     case TPM2_CC_GetCapability:
-        g_debug ("processing TPM2_CC_GetCapability");
-        response = get_cap_gen_response (resmgr, command);
+        if (!tpm2_command_has_auths(command)) {
+            g_debug ("processing TPM2_CC_GetCapability");
+            response = get_cap_gen_response (resmgr, command);
+        }
         break;
     default:
         break;
```